symai/components.py

```diff
@@ -53,27 +56,9 @@ def __init__(self, cases: List[str], default: Optional[str] = None):
         super().__init__()
         self.cases: List[str] = cases
         self.default: Optional[str] = default
-    
+
     def forward(self, sym: Symbol, *args, **kwargs) -> Symbol:
         return sym.choice(cases=self.cases, default=self.default, *args, **kwargs)
-    
-
-class Classify(Expression):
```

Comment:
Did you also trace where the Classify feature was used before removing?

Reply:
I did, that was the next thing I would have worked on, but the commit will just get too large with changing Symbia as well in here. But sure, I can do it in one go.

Reply:
Thank you. I would do it in one go to have a complete major release